test/dummy/config/boot.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'rubygems'
 ENV['BUNDLE_GEMFILE'] ||= 'Gemfile'
-gemfile = File.expand_path("../../../#{ENV['BUNDLE_GEMFILE']}", __dir__)
+gemfile = File.expand_path("../../../#{ENV.fetch('BUNDLE_GEMFILE', nil)}", __dir__)
```

Comment:
Why use `.fetch(key, nil)` when `[key]` does exactly that, more succinctly?

Reply:
Although in this case, where are ok with `nil` and the array access may look succinct, it is not expressing the developer's explicit intent (opt-in), and instead of having two variations of accessing ENV variables, rubocop has opted for this.

This is coming from one of the default rubocop conventions:

<img width="1358" alt="image" src="https://github.com/Invoca/large_text_field/assets/117316932/8aeac154-a3ec-44f4-8ca1-fe41a98662df">

More info: https://msp-greg.github.io/rubocop/RuboCop/Cop/Style/FetchEnvVar.html

If you feel strongly about it, I can revert the change, but instead of globally disabling the cop, I can disable the check for this line. Let me know your thoughts.

Reply:
I have a simple test for Rubocop's opinions:

1. Do they match opinions we'd already been expressing in our PRs/from our Style Guide? or
2. Do they point out a bug pattern we didn't know about? And hence we'll watch for in our PRs from here on out?

This one fails both tests, so please disable it globally.